### PR TITLE
Add Constructor of Color as Java have. Solve #7008

### PR DESCRIPTION
**Pull request by HydrolienF**

### CODE REVIEW

File: gdx/src/com/badlogic/gdx/graphics/Color.java [hunk 1]
```diff
@@ -82,17 +82,27 @@ public Color (int rgba8888) {
...
 	}
+	
+	/** Constructor, sets the components of the color
+	 * 
+	 * @param r the red component as int in [0,255]
+	 * @param g the green component as int in [0,255]
+	 * @param b the blue component as int in [0,255]
+	 * @param a the alpha component as int in [0,255] */
+	public Color (int r, int g, int b, int a) {
+		super(r/255f, g/255f, b/255f, a/255f);
```
Comment (MobiDevelop) [inline gdx/src/com/badlogic/gdx/graphics/Color.java:104]: I assume you intended it to be `this(...);`

Comment (HydrolienF) [reply gdx/src/com/badlogic/gdx/graphics/Color.java:104]: o yes. I'm fixing it

**Comment by mgsx-dev** [comment]
it would totally break some user code (eg. `new Color(0,0,0,1)`) which is converted to floats. With this change, their color will be broken. It already breaks libgdx predefined colors (eg. Color.WHITE). You can use `Color.toIntBits` or `Color.valueOf(hexString)` instead.

**Comment by Frosty-J** [comment]
Mm, good point. I might have a few of those implicit conversions. Heck, even gdx-setup does `ScreenUtils.clear(1, 0, 0, 1)` instead of `Color.RED`.

**Comment by HydrolienF** [comment]
> it would totally break some user code (eg. `new Color(0,0,0,1)`) which is converted to floats.

Oh rigth. I understand.
Or we can check if one value is over 1 then use same int builder than Java else use float builder.

Anyway, that's not realy a major issues even if it's disturbing for beginners.

**Comment by Frosty-J** [comment]
Using ints if over 1 else floats could cause bugs. Let's say someone fades from white to black by starting at 255 and ending at 0. Most of the fade will be fine, but at 1 there will be a flash of white. If a fade occurs over a period of 1 second at a refresh rate of 60Hz, the flash would only occur in approximately 1 in 4 fades, making the bug all the more confusing.

**Pull request closed without merging**